spec/view_models/auction_status/expiring_view_model_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+describe AuctionStatus::ExpiringViewModel do
```

Comment:
I'm not sure I completely understand when some things are in modules/classes like BidPresenter::Null and this and others are not